.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,41 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    types: [opened]
+  pull_request_review:
+    types: [submitted]
+
+concurrency: pr-${{ github.ref }}
+
+jobs:
+  dependency-checks:
+    name: Dependency Checks
+
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set Branch Name
+        uses: nelonoel/branch-name@v1.0.1
+
+      - name: Set up adoptopenjdk8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt-hotspot'
+          cache: 'gradle'
+
+      - name: Gradle Build
+        run: |
+          chmod +x ./gradlew
+          ./gradlew clean build shadowJar --info
```

Comment:
I think this needs to also `publishToMavenLocal`.  And same as above about `--info`.

Reply:
There's also a `chmod` line here

Reply:
Yes, @David-N-Perkins is correct about the task.  Should be `publishToMavenLocal`.